docs/articles/documentation/guides/basic-guides/select-page-elements.md

```diff
@@ -157,6 +157,10 @@ const sel = Selector('div').child();
 </html>
 ```
 
+If a selector matches multiple elements, only the first matching element will be used for [actions](./interact-with-the-page.md) or when retrieving [DOM Node state](../../reference/test-api/domnodestate.md). Use [.nth](../../reference/test-api/selector/nth.md) method to enumerate through other matching elements.
```

Comment:
`Use [.nth](../../reference/test-api/selector/nth.md) method to enumerate through other matching elements.`

"enumerate through" is not correct. Are you using "enumerate" in the context of "count" or "list". Or do you actually mean "iterate through"?

Reply:
Yes, iterate is the word.

Reply:
```suggestion
If a selector matches multiple elements, only the first matching element will be used for [actions](./interact-with-the-page.md) or when retrieving the [DOM Node state](../../reference/test-api/domnodestate.md). Use the [.nth](../../reference/test-api/selector/nth.md) method to iterate through other matching elements.
```